core/async/src/messaging.rs

```diff
@@ -7,7 +7,16 @@ use std::fmt::{Debug, Display};
 use std::sync::Arc;
 use tokio::sync::oneshot;
 
-/// Trait for sending a typed message.
+/// Trait for handling a message.
+/// This works in unison with the [`CanSend`] trait. An actor implements the Handler trait for all
+/// messages it would like to handle, while the CanSend trait implements the logic to send the
+/// message to the actor. Handle and CanSend do not need to be implemented by the same struct.
```

Comment:
Should we rather say, Handler and CanSend are typically not both implemented by the same struct?

Reply:
Will do!